slatedb/src/mem_table_flush.rs

```diff
@@ -147,6 +147,10 @@ impl MemtableFlusher {
                         modifier.state.manifest.core.last_l0_seq = seq;
                     };
 
+                    // update all the sequence numbers in the manifest
+                    let seq_tracker = self.db_inner.wal_buffer.seq_tracker();
```

Comment:
I'm curious about how this behaves when the WAL is disabled. The only place I see seq_tracker.insert is in the `WalBuffer`. I did see you added a test for `wal_enabled=false`, so I assume it works? (I haven't read over the test in detail). Perhaps the seqnum is applied elsewhere in a previous PR?

Reply:
I was also pretty confused about this, but the `WalBufferManager` is actually used to track sequence numbers even if the wal is disabled... Probably a little misleading, but that's a story for another time! https://github.com/slatedb/slatedb/blob/9931661f1e2ee84aab4f9f64cb1ad1d181635146/slatedb/src/batch_write.rs#L87-L104

cc @flaneur2020